core/src/main/resources/templates/email/messages.properties

```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Copyright (c) 2017 Stichting Yona Foundation
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+###############################################################################
+buddy-invitation-subject=Yona buddy invitation from {0}
```

Comment:
I'm not particularly happy with it either, but this is the normal way for Thymeleaf.